bin/combinetf2_fit.py

```diff
@@ -212,19 +214,15 @@ def make_parser():
         help="run fit on pseudo data with the given name",
     )
     parser.add_argument(
-        "--normalize",
-        default=False,
-        action="store_true",
-        help="Normalize prediction and systematic uncertainties to the overall event yield in data",
-    )
-    parser.add_argument(
-        "--project",
+        "-m",
+        "--physicsModel",
         nargs="+",
         action="append",
         default=[],
```

Comment:
Does it make more sense to just make the default argument ["basemodel"] rather than appending this to the beginning of the list later on in the code?

Reply:
Done